### PR TITLE
Sidebar width

### PR DESCRIPTION
**Pull request by adiraomj**
Fixed bug #8 

Made changes to layout SCSS. 
Updated settings CSS.

@Misplon please test. Thanks.

### CODE REVIEW

File: sass/site/_layout.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
 .corp-container {
 	margin: 0 auto;
-	max-width: none; 
+	max-width: none;
 	padding: 0 15px;
 	position: relative;
-	width: 1170px; // 1140px container.
-	
+	width: 1170px; // 1170px container.
```
Comment (Misplon) [inline sass/site/_layout.scss]: The initial comment is correct. The actual size given the box model is 1140.

File: sass/site/_layout.scss [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+
 	.responsive & {
-		max-width: 1170px; // 1140px container.
-		width: 100%; 
+		max-width: 1170px; // 1170px container.
```
Comment (Misplon) [inline sass/site/_layout.scss]: The initial comment is correct. The actual size given the box model is 1140.

File: sass/site/_layout.scss [hunk 3]
```diff
@@ -21,41 +21,66 @@
...
+
 // No Sidebar page layout setting.
 .page-layout-no-sidebar .site-content .content-area {
 	float: none;
-	width: 100%; 
+	width: 100%;
+	margin: 0;
+
+	.site-main {
+		margin: 0;
```
Comment (Misplon) [inline sass/site/_layout.scss]: Any idea why this is necessary? When is the initial margin being added to `.site-main` ?

Comment (Misplon) [reply sass/site/_layout.scss]: The same question for the above rule. Is `margin: 0;` necessary in both cases? Where is the initial margin being added?

**Comment by Misplon** [comment]
Almost there. Please, test with sidebar set to left. The content now overlaps.

File: sass/site/_layout.scss [hunk 4]
```diff
@@ -21,41 +21,66 @@
...
 		border: 0;
 		padding-right: 0;
-		width: 100%; 	
+		width: 100%;
+		margin: 0;
+	}
+}
+
+.sidebar .site-main {
+	margin: 0 $sidebar__width 0 0;
```
Comment (adiraomj) [inline sass/site/_layout.scss]: `.site-main` has its margin set here.

File: sass/site/_layout.scss [hunk 5]
```diff
@@ -21,41 +21,66 @@
...
-	width: 100%; 
+	width: 100%;
 }
 
 // Content area with sidebar.
 .sidebar .content-area {
 	float: left;
-	width: 66%;	
-	
+	margin: 0 (-$sidebar__width) 0 0;
```
Comment (adiraomj) [inline sass/site/_layout.scss]: `.content-area` margin is set here.

File: sass/site/_layout.scss (hunk 1 above)
Comment (adiraomj) [reply sass/site/_layout.scss]: Ok. Making the change

**Comment by adiraomj** [comment]
@Misplon I have made the changes. please test. Thanks.

**Pull request merged**